openapi/data/fields.py

```diff
@@ -145,6 +153,26 @@ def openapi(self, prop):
         pass
 
 
+@dataclass
+class StrValidator(Validator):
+    min_length: int = 0
+    max_length: int = 0
+
+    def __call__(self, field, value, data=None):
+        value = str(value)
```

Comment:
Won't this just `repr` anything that doesn't define `__str__`?
`'<__main__.MyClass object at 0x10fbf7630>'` might be the right length...

Reply:
Could do `if isinstance(value, str)`. Do we want to support byte strings with an encoding?